articles/azure-cache-for-redis/cache-how-to-import-export-data.md

```diff
@@ -170,6 +172,12 @@ To resolve this error, start the import or export operation before 15 minutes ha
 
 Export works only with RDB files stored as page blobs. Other blob types aren't currently supported, including Blob storage accounts with hot and cool tiers. For more information, see [Azure storage account overview](../storage/common/storage-account-overview.md). If you're using an access key to authenticate a storage account, having firewall exceptions on the storage account tends to cause the import/export process to fail.
 
+### How to export if I have firewall enabled on my storage account?
+
+For firewall enabled storage accounts, we need to check “Allow Azure services on the trusted services list to access this storage account” then, use managed identity (System/User assigned) and provision Storage Blob Data Contributor RBAC role for that object ID.
```

Comment:
For firewall enabled storage accounts, you need to first enable “Allow Azure services on the trusted services list to access this storage account” using steps mentioned [here](https://learn.microsoft.com/azure/storage/common/storage-network-security?tabs=azure-portal#grant-access-from-azure-resource-instances). Then use your managed identity (System/User assigned) and assign Storage Blob Data Contributor RBAC role to your managed identity that you want to use to access storage account.

Reply:
@flang-msft Can you validate if the link url looks correct?